FpsCheck.py

```diff
@@ -35,6 +35,7 @@ class FpsCheck:
                 without any given check failing halfway through and not 
                 catching other issues. 
   """
+    WELLKNOWN = "/.well-known/first-party-set.json"
```

Comment:
If nothing outside of this file needs to use this string (e.g. no tests -- do they also repeat the well-known path?), I would make it a file-local constant, instead of a class-scoped constant. That reduces the visibility of it, which enforces more encapsulation.